cgp/local_search/random_search.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+from typing import Callable
+
+
+class RandomLocalSearch:
+    def __init__(
+        self,
+        objective: Callable,
+        seed: int,
+        n_steps: int = 10,  # todo: should this be dependent on the number of parameters?
+        permissible_values: np.ndarray = np.logspace(start=0, stop=7, num=8, base=2, dtype=int),
```

Comment:
in the current implementation, is it, for example, possible to say "parameters should be uniformly sampled in the interval [0, 1]"? if not, I think it would be a good idea to make the generation of permissible values modular to support such use cases